test/alternative_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper.rb"
+require "update-alternatives/model/alternative"
+
+describe UpdateAlternatives::Alternative do
+
+  describe ".all" do
```

Comment:
Is not about checking that the names are correctly read (which, indeed, belongs to the other test), it's more about checking that we are importing the correct set of alternatives (same number and same names).